### PR TITLE
IRSA-1032: Error message is not user friendly anymore

### PR DESCRIPTION
**Pull request by loitly**
Also fixed IRSA-782: Update Error Message in Time Series Tool

https://caltech-ipac.atlassian.net/browse/IRSA-1032
https://caltech-ipac.atlassian.net/browse/IRSA-782

http://localhost:8080/firefly/ts.html, or
http://localhost:8080/irsaviewer/timeseries

Try different variation of tables and mission to make sure that error message matches the condition of your input.

### CODE REVIEW

File: src/firefly/java/edu/caltech/ipac/firefly/server/util/ipactable/JsonTableUtil.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import edu.caltech.ipac.util.DataGroup;
 import edu.caltech.ipac.util.DataType;
 import edu.caltech.ipac.util.StringUtils;
+import org.eclipse.jetty.util.StringUtil;
```
Comment (ejoliet) [inline src/firefly/java/edu/caltech/ipac/firefly/server/util/ipactable/JsonTableUtil.java]: Are you sure you are not using the StringUtils from Firefly instead?

File: src/firefly/js/tables/ui/TablePanel.jsx [hunk 2]
```diff
@@ -313,16 +313,15 @@ function Loading({showTitle, tbl_id, title, removable, bgStatus}) {
...
-        const origRequest = omit(cloneDeep(request), 'filters', 'sortInfo');
-        dispatchTableSearch(origRequest);
+        dispatchTableSearch(JSON.parse(prevReq));
     };
     return (
         <div className='TablePanel__error'>
+            <div><b>Table Load Error:</b></div>
             <div>{message}</div>
-            {canReset && <button type='button' className='button std' onClick={reloadTable}>Reload</button>}
+            {prevReq && <button type='button' className='button std' onClick={reloadTable}>Back</button>}
```
Comment (ejoliet) [inline src/firefly/js/tables/ui/TablePanel.jsx]: What is the 'back' button doing? Don't you mean 'OK' or some user reading confirmation?

File: src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js [hunk 3]
```diff
@@ -84,7 +84,7 @@ export function isValidPTFTable() {
     }
     else {
          const errorMsg = `The uploaded table is not valid. The PTF option requires pid.
-                        Please select one of the other upload options`;
+                        Please select the "Other" upload option for tables that do not meet these requirements.`;
```
Comment (ejoliet) [inline src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js]: I think we want to keep the other options as a text and not indicating the option as alternative. The previous text was good, no need to change unless i'm missing something else.
Same for WISE case below.

Comment (ejoliet) [reply src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js]: See IRSA-782 for the exact text.

File: src/firefly/java/edu/caltech/ipac/firefly/server/util/ipactable/JsonTableUtil.java (hunk 1 above)
Comment (loitly) [reply src/firefly/java/edu/caltech/ipac/firefly/server/util/ipactable/JsonTableUtil.java]: Good eyes.  It was inserted as by IntelliJ auto-complete feature.  Not used.  But, I'll remove it.

File: src/firefly/js/tables/ui/TablePanel.jsx (hunk 2 above)
Comment (loitly) [reply src/firefly/js/tables/ui/TablePanel.jsx]: This error usually shows up when you've entered a bad filter or sort condition.  The 'Back' button is to put the table back to the state prior to this bad operation.  If you have a better word, let me know.  I have no problem changing it.

File: src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js (hunk 3 above)
Comment (loitly) [reply src/firefly/js/templates/lightcurve/ptf/PTFMissionOptions.js]: Okay, now I am confused.  The exact text in the ticket is what I put here, which is not what they want.  What they want is what we have already.  I'll just revert the change.

**Pull request merged**